tests/integration/pilot/common/routing.go

```diff
@@ -3826,6 +3833,67 @@ spec:
 	}
 }
 
+func testServiceEntryWithMultipleVIPsAndResolutionNone(t TrafficContext) {
```

Comment:
Can all the ambient tests splitting out the SE to v4 and v6 (`dummy-v4.example.com`) be joined now? or I think we need to do the same logic for routes as well probably